airbyte-webapp/src/components/JobItem/components/JobLogs.tsx

```diff
@@ -27,11 +24,21 @@ const JobLogs: React.FC<IProps> = ({ id, jobIsFailed }) => {
     return <Logs />;
   }
 
-  const data = job.attempts.map((item, index) => ({
+  const currentAttempt = job.attempts[attemptNumber].attempt;
+  const logs = job.attempts[attemptNumber]?.logs;
+  const path = [
+    "/tmp/workspace",
+    id,
+    currentAttempt.id,
+    "airbyte-logs.log",
```

Comment:
```suggestion
    "logs.log",
```

Reply:
This one is just UI change that will be displayed when a user checks logs or the path of the file to which the user tries to save.

I thought that we may want to select a better name. Should we keep the one that you proposed?

Reply:
The problem is that this corresponds to an actual path from the perspective of the container. So the actual path is `logs.log` instead of `airbyte-logs.log`.

Reply:
Actually in the very near future this is going to be generated by logs from a couple of different sources. It's probably best to remove it since it isn't accurate in all cases.